templates/chat/README.md

```diff
@@ -0,0 +1,76 @@
+<div alt style="text-align: center; transform: scale(.5);">
+	<picture>
+		<source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/tldraw/tldraw/main/assets/github-hero-dark.png" />
+		<img alt="tldraw" src="https://raw.githubusercontent.com/tldraw/tldraw/main/assets/github-hero-light.png" />
+	</picture>
+</div>
+
+This repo contains a starter-kit for making an AI chat application using [tldraw](https://github.com/tldraw/tldraw).
+
+## Local development
+
+Install dependencies with `yarn` or `npm install`.
+
+Run the development server with `yarn dev` or `npm run dev`.
+
+Open `http://localhost:3000/` in your browser to see the app.
+
+## Overview
+
+This starter kit demonstrates how to build an AI chat application that uses tldraw to provide sketches and annotated image to the model. The app features:
+
+- Integrated whiteboard for providing visual context
+- Image annotation and markup
+- Easy switching between text chat and visual canvas input
+
+Key interactions include:
+
+- Chat with AI using natural language
+- Click the whiteboard button to open the tldraw canvas
+- Draw, sketch, and create diagrams to supplement conversations
+- Annotate images and visual content directly on the canvas
+
+## Environment Setup
+
+Create a `.env.local` file in the root directory and add your OpenAI API key:
+
+```
+OPENAI_API_KEY=your_openai_api_key_here
```

Comment:
b/c i'm a lazy eng i loaded up the template without setting this (knowing that i prbly needed to).
i was surprised that there's no error/or toast in the UI to notify that the key is missing. prbly some feedback to the dev that the key should be set?